meerkat_abacus/config.py

```diff
@@ -104,6 +104,10 @@ def __init__(self):
         elif self.initial_data_source == "AWS_S3":
             self.get_data_from_s3 = 1  # int(os.environ.get("GET_DATA_FROM_S3", False))
             self.initial_data = "S3"
+        else:
+            raise ValueError(
+                f"INITIAL_DATA_SOURCE = {self.initial_data_source} unsupported."
+            )
```

Comment:
Could you keep the parenthesis in shape please?
```
else:
  raise ValueError(f"...")
```

Reply:
Or to have it < 80 chars:
```
        else:
            msg = f"STREAM_DATA_SOURCE = {self.stream_data_source} unsupported."
            raise ValueError(msg)
```

Reply:
Done :+1: